src/nupic/algorithms/Svm.hpp

```diff
@@ -905,10 +908,10 @@ namespace nupic {
 		   int cache_size =100,
 		   int shrinking =1,
 		   bool probability =false,
-		   int seed =-1)
+		   int seed =0)
 	  : param_(kernel, probability, gamma, C, eps, cache_size, shrinking),
 	    problem_(new problem_type(n_dims, true, threshold)),
-	    model_(nullptr), rng_(seed != -1 ? seed : 0),
+	    model_(nullptr), rng_(seed != RANDOM_SEED ? seed : 0),
```

Comment:
@scottpurdy this looks little unfortunate, we decided to use default fixed seed (=0) everywhere; -1 denotes random seed should be chosen. And then the `rng` uses (internally) 0 for random seed?